operator/api/v1/csiscaleoperator_types.go

```diff
@@ -315,7 +315,7 @@ type CSICluster struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Secrets",xDescriptors="urn:alm:descriptor:io.kubernetes:Secret"
 	Secrets string `json:"secrets"` // TODO: Secrets should be Singular
 
-	// secureSslMode specifies if a secure SSL connection to connect to GPFS is required.
+	// secureSslMode specifies if a secure SSL connection to connect to spectrum scale is required.
```

Comment:
"to spectrum scale" replace with "to Spectrum Scale cluster"

Reply:
Fixed.